ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesDocuments/SeriesDocuments.js

```diff
@@ -64,25 +64,24 @@ export default class SeriesDocuments extends Component {
       { title: 'Volume', formatter: this.volumeFormatter },
       { title: 'ID', field: 'metadata.pid' },
     ];
-    const hasDocuments = !isEmpty(seriesDocuments);
     return (
       <Loader isLoading={isLoading}>
         <Error error={error}>
-          {hasDocuments ? (
-            <ResultsTable
-              data={seriesDocuments.hits}
-              columns={columns}
-              totalHitsCount={seriesDocuments.total}
-              name={'documents'}
-              seeAllComponent={this.seeAllButton()}
-              showMaxRows={showMaxDocuments}
-            />
-          ) : (
-            <InfoMessage
-              header="No multiparts found."
-              content={'Start with a multipart to attach it to this series.'}
-            />
-          )}
+          <ResultsTable
+            data={seriesDocuments.hits}
+            columns={columns}
+            totalHitsCount={seriesDocuments.total}
+            name={'documents'}
+            seeAllComponent={this.seeAllButton()}
+            showMaxRows={showMaxDocuments}
+            renderEmptyResultsElement={() => (
+              <InfoMessage
+                header="No volumes in this series."
+                content="Start with a book/article to attach it to this series."
```

Comment:
minor: WDYT with: start **from** a ...

Reply:
👍 maybe you have an idea for something more specific. I wanted to express to start from book/article **details page** but I'm not sure if it is clear for the user